Tests/AppTests/RouteTests.swift

```diff
@@ -5,35 +5,54 @@ import HTTP
 @testable import Vapor
 @testable import App
 
-/// This file shows an example of testing 
-/// routes through the Droplet.
-
 class RouteTests: TestCase {
-    let drop = try! Droplet.testable()
-    
-    func testHello() throws {
-        try drop
-            .testResponse(to: .get, at: "hello")
-            .assertStatus(is: .ok)
-            .assertJSON("hello", equals: "world")
-    }
-
-    func testInfo() throws {
-        try drop
-            .testResponse(to: .get, at: "info")
-            .assertStatus(is: .ok)
-            .assertBody(contains: "0.0.0.0")
-    }
+  let drop = try! Droplet.testable()
```

Comment:
lets move the middleware route tests to ClientMiddlewareTests.swif

Reply:
Discussed. Leave it as is.